SCOPES.md

```diff
@@ -0,0 +1,38 @@
+# Prefixes
+Supported scope prefixes:
+* `project:releng:snapcraft`
+  * Used to tell which Google Play store a given apk should update.
+
+# Scopes
+* `{scope_prefix}:firefox:beta`
+  * Uploads to the *beta* snap track.
+    * Uses branding "Firefox Beta"
+  * **Conflicts**: with any other `{scope_prefix}:firefox:*`
+  * **Branch Restrictions**:
+    * `beta`
+
+* `{scope_prefix}:firefox:candidate`
+  * Uploads to the *candidate* snap track.
+    * Uses branding "Firefox"
+  * **Conflicts**: with any other `{scope_prefix}:firefox:*`
+  * **Branch Restrictions**:
+    * `release`
+
+* `{scope_prefix}:firefox:esr`
+  * Uploads to the ???.
```

Comment:
`to the esr (esr/stable) snap track.`. On https://snapcraft.io/firefox, click on "Install..." and see the esr track in the dropdown menu.